File: Chavez-Rubio-Javier/P1/C/execl.c
#include <stdio.h>
#include <stdlib.h>
#include <sys/types.h>

int main(){
    int ret;
    char *arg1 = "1234567890";
	char *pathname = "hello";

    ret = execl(pathname, pathname, arg1, NULL);

    return 0;
}  

File: Chavez-Rubio-Javier/P2/Python/execl.py
# $ python3 execl.py

import subprocess

arg1 = "1234567890"
pathname = "./hello"

result = subprocess.run([pathname, pathname, arg1], capture_output=True)

if result.returncode == 0:
    print(result.stdout.decode())
else:
    print(f"Error: {result.stderr.decode()}")
